vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -56,37 +68,56 @@ function Get-FilesToCacheOnVHD
             "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1",
-            "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg"
+            "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
+            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.2.zip"
         );
+        "c:\akse-cache\containerd\" = @(
+            $global:containerdPackageUrl
+        );
+
         "c:\akse-cache\win-k8s\" = @(
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.7-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.4-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.5-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.16.1-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.14.7-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.14.8-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.15.4-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.15.5-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.15.7-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.16.1-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.16.2-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.16.4-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/v1.17.0-1int.zip"
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.10-azs/windowszip/v1.15.10-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.11-azs/windowszip/v1.15.11-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-azs/windowszip/v1.15.12-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.13-azs/windowszip/v1.16.13-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.9-azs/windowszip/v1.17.9-azs-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.10/windowszip/v1.15.10-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.11/windowszip/v1.15.11-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.11-hotfix.20200714/windowszip/v1.15.11-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12/windowszip/v1.15.12-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-hotfix.20200714/windowszip/v1.15.12-hotfix.20200714-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-hotfix.20200623/windowszip/v1.15.12-hotfix.20200623-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.11/windowszip/v1.16.11-1int.zip",
```

Comment:
Need to add 1.16.10 since aks uses it.

Reply:
v1.16.10-hotfix.20200714

Reply:
Good catch. Thanks.

Reply:
The windows supported versions in AKS, v1.16.10, v1.15.11 and v1.15.12 are all included here.